internal/controllers/submariner/loadbalancer_resources.go

```diff
@@ -104,6 +106,8 @@ func newLoadBalancerService(instance *v1alpha1.Submariner, platformTypeOCP strin
 			"service.kubernetes.io/ibm-load-balancer-cloud-provider-ip-type":                   "public",
 			"service.kubernetes.io/ibm-load-balancer-cloud-provider-vpc-health-check-protocol": "http",
 		}
+	case string(configv1.KubevirtPlatformType):
```

Comment:
Does setting externalTrafficPolicy to corev1.ServiceExternalTrafficPolicyTypeCluster will work for both kubevirt and HCP Kubevirt clusters ?  or it is needed only for HCP

Reply:
In this particular case IP address was preserved, maybe we should check the setup again. There were two level of load balancer service one created in the HCP cluster and in response to that one will be created in the hosted cluster.

The platform type was kubevirt , do we have another way to identify kubevirt vs HCP kubevirt?